### PR TITLE
feat(): Android SDK to load WebSDK for display ads.

### PR DESCRIPTION
**Pull request by ari-ada**
Android SDK to load WebSDK for display ads.

### CODE REVIEW

File: superawesome-base/src/main/java/tv/superawesome/sdk/publisher/managed/SAManagedBannerAd.kt [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+    fun JSONObject.toQueryParams(): String {
+        var queryParams = ""
+
+        var looper = this.keys()
+        while(looper.hasNext()) {
+            val key = looper.next()
+            if (key.equals("test") && this.get(key).toString().equals("false")) {
+                continue
+            }
```
Comment (ari-ada) [inline superawesome-base/src/main/java/tv/superawesome/sdk/publisher/managed/SAManagedBannerAd.kt:89]: We found a bug in the ads-server. If you pass `false` then ads-server won't return. Will fix the ads-server but those lines won't affect anything else.

**Comment by ari-ada** [comment]
the Activity does not need to be merged. I have prepared it for the green team for testing but it is not needed afterwards.

File: app/src/main/java/tv/superawesome/demoapp/MainActivity.java [hunk 2]
```diff
@@ -39,67 +41,25 @@
...
-        SAVideoAd.enableBackButton();
-        SAVideoAd.setListener((SAInterface) (placementId, event) -> {
-            Log.d("SADefaults", "VIDEO AD: " + placementId + " -> Event : " + event);
-
-            if (event == SAEvent.adLoaded) {
-//                SAVideoAd.play(placementId, MainActivity.this);
-            }
+        final SAManagedBannerAd managedBannerAd = findViewById(R.id.preview);
+        managedBannerAd.setListener((SAInterface) (placementId, event) -> {
+            managedBannerAd.load(placementId);
```
Comment (devgabrielcoman) [inline app/src/main/java/tv/superawesome/demoapp/MainActivity.java:48]: is this load or play?

File: build.gradle [hunk 3]
```diff
@@ -1,4 +1,5 @@
 buildscript {
+    ext.kotlin_version = '1.3.71'
```
Comment (devgabrielcoman) [inline build.gradle:2]: you're  possibly running two kotlin versions here? stick with the one that says `1.3.72`, it's very stable. Unless you have a reason to switch to `1.3.71`

**Review by devgabrielcoman — APPROVED** [review]
Looks okay @ari-ada plus/minus a few comments

**Comment by ari-ada** [comment]
ok. We went through all problems with Gabriel and Nicolas. Will deal with merge conflicts and merge

**Pull request merged**